ssl4pl.c

```diff
@@ -4252,7 +4252,7 @@ pl_ssl_session(term_t stream_t, term_t session_t)
        !(session = SSL_get1_session(ssl)) )
     return PL_existence_error("ssl_session", stream_t);
 
-#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
This code used to compile with older versions of libressl.  So, I _think_ the correct rule is probably as below.   Do you have the version number of this change (and is the version format the same as for openssl?)

     #if OPENSSL_VERSION_NUMBER < 0x10100000L || LIBRESSL_VERSION_NUMBER < something